### PR TITLE
Removal of Plot Config

### LINKED ISSUE #553: PlotConfig constructor / removal of plotconfig

### PR DESCRIPTION
**Pull request by xjules**
**Issue**
Resolves #553

Semantically PlotConfig does not belong to libres and it's not much used besides ert.
**Pre un-WIP checklist**
- [ ] Have completed graphical integration test steps

**Depends on**
* Equinor/libecl#

### CODE REVIEW

**Comment by andreabrambilla** [comment]
jenkins build me

**Comment by andreabrambilla** [comment]
test this please

File: python/res/enkf/config_keys.py [hunk 1]
```diff
@@ -85,7 +89,11 @@ class ConfigKeys:
     OPTION           = "OPTION"
     VALUE            = "VALUE"
     INSTALL_JOB      = _install_job_key()
-    PLOT_SETTINGS    = _plot_settings_key()
+    PLOT_SETTINGS_KEY= _plot_settings_key()
```
Comment (ManInFez) [inline python/res/enkf/config_keys.py]: Are these used anywhere?

File: python/tests/res/enkf/test_programmatic_res_config.py [hunk 2]
```diff
@@ -574,10 +571,7 @@ def test_large_config(self):
             self.assert_equal_ert_workflow(loaded_res_config.ert_workflow_list,
                                            prog_res_config.ert_workflow_list)
 
-            self.assert_equal_plot_config(loaded_res_config.plot_config,
-                                          prog_res_config.plot_config)
-
-            self.assertEqual(0, len(prog_res_config.failed_keys))
+            # self.assertEqual(0, len(prog_res_config.failed_keys))
```
Comment (ManInFez) [inline python/tests/res/enkf/test_programmatic_res_config.py]: Remove commented line

File: python/tests/res/enkf/test_programmatic_res_config.py [hunk 3]
```diff
@@ -213,6 +213,7 @@ def setUp(self):
                                   "OBS_CONFIG"       : "../input/observations/obsfiles/observations.txt",
                                   "TIME_MAP"         : "../input/refcase/time_map.txt",
 
+                                  # "MARKUS" :
```
Comment (ManInFez) [inline python/tests/res/enkf/test_programmatic_res_config.py]: Remove commented line

File: python/tests/res/enkf/test_enkf.py [hunk 4]
```diff
@@ -173,7 +173,7 @@ def test_config( self ):
             #self.assertIsInstance(main.local_config(), LocalConfig) #warn: Should this be None?
             self.assertIsInstance(main.siteConfig(), SiteConfig)
             self.assertIsInstance(main.eclConfig(), EclConfig)
-            self.assertIsInstance(main.plotConfig(), PlotSettings)
+            #self.assertIsInstance(main.plotConfig(), PlotSettings) removed PlotSettings
```
Comment (ManInFez) [inline python/tests/res/enkf/test_enkf.py]: Remove commented line

File: python/res/enkf/config_keys.py (hunk 1 above)
Comment (xjules) [reply python/res/enkf/config_keys.py]: Yes, removing it now.

File: python/tests/res/enkf/test_programmatic_res_config.py (hunk 3 above)
Comment (xjules) [reply python/tests/res/enkf/test_programmatic_res_config.py]: OK

File: python/tests/res/enkf/test_enkf.py (hunk 4 above)
Comment (xjules) [reply python/tests/res/enkf/test_enkf.py]: OK

**Review by ManInFez — APPROVED** [review]
LGTM!

**Pull request merged**